lib/rules/headers/dl.js

```diff
@@ -117,7 +119,19 @@ exports.check = function (sr, done) {
         if (matches) {
             sn = matches[1];
             previousURI = $linkPrev.text();
-            if (sn !== shortname) sr.error(self, "this-previous-shortname");
+            if (sn !== shortname) {
+              fetch(previousURI).then(function(body) {
```

Comment:
@tripu if the shortname differs (`sn !== shortname`), you should follow the latest version link, and not the previous version, then if it gives a 404, send a warning instead of an error.
```
<denis> if you want to do it the easy way, you can do that
<denis> - compare shortname from this version and previous version
<denis> - if they are different, follow the shortlink and if it's a 404, there's good chance it's shortname change
<tripu> ...in which case we show a warning
<tripu> (not an error)
<tripu> ?
<denis> yes
<tripu> ok
```